src/main/java/jenkins/plugins/jobcacher/ArbitraryFileCache.java

```diff
@@ -330,7 +330,8 @@ public enum CompressionMethod {
 
         NONE(new SimpleArbitraryFileCacheStrategy()),
         ZIP(new ZipArbitraryFileCacheStrategy()),
-        TARGZ(new TarGzArbitraryFileCacheStrategy());
+        TARGZ(new TarArbitraryFileCacheStrategy("gz", ".tgz")),
```

Comment:
`.tar.gz` might also be used here for consistency with `.tar.zst`.

Reply:
Yes, I thought about this, but as this would break existing caches, I decided to not rename it.